### PR TITLE
Add Meta announcement

### PR DESCRIPTION
**Pull request by wipfli**
Adds a news article about the renewed sponsorship from Meta.

@birkskyum do you have a dark version of the new logo? Also, where is our official logo SVG stored?

### CODE REVIEW

**Comment by birkskyum** [comment]
Yes, for light backgrounds there is one here. They are all stored in the Figma i linked.

maplibre-light-bg

**Comment by wipfli** [comment]
Tweet will be "Meta continues to support MapLibre and increases its donation from the Silver to the Gold tier in its second year of sponsorship. Read more..."

**Review by louwers — APPROVED** [review]
Consider adding a link to the pre-release. Otherwise looks good!

File: content/news/2023-11-08-meta-continues-to-support-maplibre/index.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+Meta continues to support MapLibre and increases its donation from the Silver to the Gold tier in its second year of sponsorship.
+
+<img src="maplibre-meta-logo.png" width="100%"/>
+
+<br />
+<br />
+
+The Gold tier donation of USD 150,000 will be used to 25 percent for development of new features and to 75 percent for general maintenance of the MapLibre codebases and project coordination.
+
+But not only is Meta supporting MapLibre financially. In 2023, Meta contributed a substantial amount of engineering time to the Metal project team in MapLibre Native. The fact that today we have a working early-access prerelease for Metal is in large part thanks to the excellence, team play, and graphics expertise that Meta provided.
```
Comment (wipfli) [inline content/news/2023-11-08-meta-continues-to-support-maplibre/index.md:18]:
```suggestion
But not only is Meta supporting MapLibre financially. In 2023, Meta contributed a substantial amount of engineering time to the Metal project team in MapLibre Native. The fact that today we have a working early-access [prerelease](https://github.com/maplibre/maplibre-native/issues/1609) for Metal is in large part thanks to the excellence, team play, and graphics expertise that Meta provided.
```

**Pull request merged**